var/spack/repos/builtin/packages/py-mpldock/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMpldock(PythonPackage):
+    """Dock matplotlib figures and other widgets."""
+
+    homepage = "https://github.com/peper0/mpldock"
+    pypi = "mpldock/mpldock-0.1.tar.gz"
+
+    maintainers("LydDeb")
+
+    version("0.1", sha256="8fee2e9cb25e122c9e11c15ea3ad22c9d03e78a08a0bc2b7f453230d58a31a55")
+
+    depends_on("python@3.6:3.10", type=("build", "run"))
```

Comment:
Where is this upper bound coming from? For me it builds fine python@3.11 (import tests also work)

Reply:
This comes from the `classifier` inside `setup.py`. I remove the version constraint on python because the `classifier` is just a comment.